proto/viam/app/v1/app.proto

```diff
@@ -858,6 +860,7 @@ message ListFragmentsResponse {
 
 message GetFragmentRequest {
   string id = 1;
+  string organization_id = 2;
```

Comment:
Should this be marked optional, to avoid a breaking request change?

Reply:
adding new fields should be [backwards compatible](APP-7107/ehhong/list-fragment-machines-ff)

Reply:
Gotcha, didn't realize that if a client failed to provide the field then an empty string would be used

Reply:
Adding fields is not a breaking change! Currently this field would go unused in the handler. I don't know if adding this would change behavior for any users outside of App.